src/llmcompressor/pipelines/data_free/pipeline.py

```diff
@@ -27,5 +28,9 @@ def __call__(
         :param dataloader: loads data for calibration
         :param dataset_args: dataset arguments relevant to pipelines
         """
+        # some ops are still performed on the model by modifiers
+        # we want those ops to occur on the GPU
+        dispatch_for_generation(model)
```

Comment:
That means we can leverage more than one gpu for data free cases, including weight-only RTN schemes?

Reply:
Technically yes, although the weights are still calibrated in a synchronous `for` loop, so there's no speedup gained from the extra gpus